src/quacc/calculators/mrcc/mrcc.py

```
@@ -0,0 +1,147 @@
+        parameters = dict(parameters)
+
+        mrccinput = {"calc": "PBE", "basis": "def2-SVP"}
+
+        kw = {"charge": 0, "mult": 1, "mrccinput": mrccinput, "mrccblocks": ""}
+        kw.update(parameters)
+
+        io.write_mrcc(directory / self.inputname, atoms, kw)
+
+    def read_results(self, directory):
+        return io.read_mrcc_outputs(directory, directory / self.outputname)
+
+    def load_profile(self, cfg, **kwargs):
+        return MrccProfile.from_config(cfg, self.name, **kwargs)
+
+
+class MRCC(GenericFileIOCalculator):
+    """Class for doing MRCC calculations.
+
+    Example:
+
+      calc = MRCC(charge=0, mult=1, mrccinput={'calc':PBE, 'basis':'def2-SVP'},
+        mrccblocks=' ')
+    """
+
+    def __init__(
+        self,
+        *,
+        profile=None,
+        directory=".",
+        parallel_info=None,
+        parallel=None,
+        **kwargs,
+    ):
+        """Construct MRCC-calculator object.
+
+        Parameters
+        ==========
+        charge: int
+
+        mult: int
+
+        mrccinput : dict[str,str]
+
+        mrccblocks: str
+
+
+        Examples
+        ========
+        Use default values:
+
+        >>> from quacc.calculators.mrcc.mrcc import MRCC
+        >>> MyMrccProfile = MrccProfile("/path/to/mrcc/dir/dmrcc")
+        >>> h = Atoms(
+        ...     "H",
+        ...     calculator=MRCC(
+        ...         profile=MyMrccProfile,
+        ...         charge=0,
+        ...         mult=1,
+        ...         directory="water",
+        ...         mrccinput={"calc": PBE, "basis": "def2-SVP"},
+        ...         mrccblocks=" ",
+        ...     ),
+        ... )
+
+        """
+
+        assert (
+            parallel is None
+        ), "MRCC does not support keyword parallel - use mrccblocks or mrccinput"
+        assert (
+            parallel_info is None
+        ), "MRCC does not support keyword parallel_info - use mrccblocks or mrccinput"
```

This was bad practice in ASE: https://gitlab.com/ase/ase/-/issues/1356. Instead, one should raise a specific error. In any case, these keyword arguments are no longer supported by ASE anyway.